dev/src/main/resources/messages/ui/CodewindUIBundle.properties

```diff
@@ -243,8 +243,10 @@ DialogYesButton=Yes
 DialogNoButton=No
 DialogCancelButton=Cancel
 
-ProjectNotImportedDialogTitle=The {0} project is not imported into Eclipse
-ProjectNotImportedDialogMsg=There is no source available for debugging because the project has not been imported. Import the {0} project?
+ProjectMustBeOpenedToDebugTitle=Project must be opened
+ProjectMustBeOpenedToDebugMessage=To perform the selected action, the project must first be opened in a window. Continue?
+ProjectIsOpenedInAnotherWindowTitle=Select other window
+ProjectIsOpenedInAnotherWindowMsg=The project is already opened in another window.  Select it and continue?
```

Comment:
- For the title, maybe, "Select project window"? (In case there is more than one other window open.)
- For the message, maybe, "The project is open in another window. Select the window and continue." (I'm thinking you wouldn't need the question mark because they need to select the project window in order to continue, right?)
- Also, only need one space after a period at the end of a sentence.